package.json

```diff
@@ -38,7 +38,7 @@
     "clean": "gts clean",
     "compile": "tsc -p . && cp -r protos build/",
     "compile-protos": "compileProtos src",
-    "prepare": "npm run compile-protos && npm run compile && rm ./build/package.json",
+    "prepare": "npm run compile-protos && npm run compile",
```

Comment:
I think you would need to revert the change from `require` to `import` approach for loading `package.json`. I don't think removing this is enough